Artifacts/windows-iis/artifact.ps1

```diff
@@ -0,0 +1,37 @@
+$artifactLogFolder = Join-path -path $env:ProgramData -childPath 'DTLArt_IIS'
+New-Item -ItemType Directory -Force -Path $artifactLogFolder
+
+$winVerString = (Get-WmiObject Win32_OperatingSystem).Caption
+Write-Output "Detected Windows Version: $winVerString"
+$scriptFolder = Split-Path $Script:MyInvocation.MyCommand.Path
+
+if ($winVerString -like 'Microsoft Windows Server 2008 R2*' -or $winVerString -like 'Microsoft Windows Server 2012*'){
+    Write-Output 'Installing IIS on Windows Server 2008 R2 or 2012'
+    $localLog = Join-Path $artifactLogFolder 'IISArtifact.log'
+    $pkgMgrExe = join-path $env:windir system32\pkgmgr.exe
+    $packages = 'IIS-WebServerRole;IIS-WebServer;IIS-IIS6ManagementCompatibility;IIS-Metabase;IIS-WMICompatibility;IIS-LegacyScripts;IIS-LegacySnapIn'
+    $pkgMgrArgs = "/l:$localLog /iu:$packages"
+    $pkgMgrProcess = Start-Process -FilePath $pkgMgrExe -ArgumentList $pkgMgrArgs -Wait -PassThru
+
+    if($pkgMgrProcess.ExitCode -eq 0){
+        Write-Output 'Successfully installed IIS'
+    }
+    elseif($pkgMgrProcess.ExitCode -eq 3010){
+        Write-Output 'IIS install is finished. A reboot is required'
```

Comment:
Minor, missing period after `A reboot is required`.

Reply:
😄